src/App.tsx

```diff
@@ -1,61 +1,105 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+
+  const [selectedUser, setSelectedUser] = useState(0);
+
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const [inputText, setInputText] = useState('');
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setTitleError(false);
+    setUserError(false);
+
+    if (!inputText.trim()) {
+      setTitleError(true);
+    }
+
+    if (!selectedUser) {
+      setUserError(true);
+    }
+
+    if (inputText.trim() && selectedUser !== 0) {
+      const newId =
+        todos.length > 0 ? Math.max(...todos.map(todo => todo.id)) + 1 : 1;
+
+      const newTodo = {
+        id: newId,
+        title: inputText,
+        completed: false,
+        userId: selectedUser,
+      };
+
+      setTodos([...todos, newTodo]);
+      setInputText('');
+      setSelectedUser(0);
+    }
+  };
+
+  function handleInput(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
It is important to follow the same way of creating functions everywhere and more descriptive name
```suggestion
  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
```